conf/nginx.dev.conf

```diff
@@ -44,6 +44,16 @@ http {
             proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
         }
 
+        location /api/feedback {
+            rewrite /api/(.*) /$1 break;
+
+            proxy_pass http://feedback-service;
+            proxy_set_header Host $host;
+            proxy_set_header X-Real-IP $remote_addr;
+            proxy_redirect off;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        }
+
```

Comment:
NGinX bullied a lot, but nice job on getting it to work in the end! 😄 
No other remarks here.